### ISSUE TITLE
Stuck in redstone area

### ISSUE THREAD

**Issue by Nuratai** [Wynncraft/Issues#8887]
I am currently trapped inside of a Redstone Contraption underneath the Exit from the Potion Merchant mini-dungeon near the starting city. I had gotten stuck somehow on the door in it, and when I triple right clicked to attempt to teleport ahead, it teleported me into the redstone area. I have attempted asking other players if I can trade them more emeralds than what it costs to buy a teleportation ticket, but nobody will respond. Please help!

![glitch](https://user-images.githubusercontent.com/29028954/30011606-2f55bb24-90fe-11e7-9113-68f0c82779ac.png)

**Comment by ittnoraA** [comment]
Heyo! When you have 5 or more soul points, you can do `/kill` to respawn in the nearest city you've visited before.

**Comment by Nuratai** [comment]
Yea, I ended up doing that. The main issue was another player had activated
both an XP and Loot bomb after I had gotten stuck, and i didn't get out
until over an hour later when I decided to just /kill. I didn't lose any of
my unidentified items, but I lost a rank 2 thunder powder, as well as
missing the bombs. Now I'm just staying around the secomd city(where those
accursed mooshrooms are)and grinding levels.
